packages/pigeon/CHANGELOG.md

```diff
@@ -1,5 +1,6 @@
 ## 22.7.1
```

Comment:
You'll need to add this change to the pubspec.yaml file, and the generator_tools.dart file.

Reply:
Done